docs/user-guide/system-admin/configuration/document-storage-directory.md

```diff
@@ -1,31 +1,27 @@
+Zanata will store files for some uploaded documents on the file system in a configured directory. This directory must be configured by using a system property. Zanata will create the directory if it does not yet exist.
 
-Since files were previously stored as BLOBs in the database, there is a migration operation to move the files from the database to the file system. Migration is performed when liquibase is run, which will happen automatically at server startup, or can be initiated from the command line. If liquibase is run from the command line, the directory must be passed as a parameter to liquibase, in addition to the JNDI name binding.
+Since files were previously stored as BLOBs in the database, there is a migration operation to move the files from the database to the file system. Migration is performed when liquibase is run, which will happen automatically at server startup, or can be initiated from the command line. If liquibase is run from the command line, the directory must be passed as a parameter to liquibase, in addition to the system property.
 
 ## Configuring the Directory (JNDI)
-The path for document storage should be bound to JNDI name ```java:global/zanata/files/document-storage-directory```
+The path for document storage should be set using a system property when starting JBoss.
 
 ### Detailed Procedure
-Open `$JBOSS_HOME/standalone/configuration/standalone.xml` and look for the `naming` subsystem. Add the following section to the `bindings` section:
+Open `$JBOSS_HOME/standalone/configuration/standalone.xml` and look for the `system-properties` section. Add the following property declaration:
 
 ```xml
-<subsystem xmlns="urn:jboss:domain:naming:1.3">
-    ...
-    <bindings>
-        <simple name="java:global/zanata/files/document-storage-directory" value="/example/path"/>
-    </bindings>
-    ...
-</subsystem>
+<property name="zanata.files.documentstorage.directory" value="/example/path"/>
 ```
 
+Alternatively, you can pass this and other system properties to JBoss when starting it (see JBoss documentation for details on how to do this).
+
 ## Running Liquibase from command line
-This section will not explain how to run liquibase from the command line, just how to add the required parameter. The parameter is ```document.storage.directory``` and should be added to the liquibase command in the form ```-Ddocument.storage.directory=/example/path```. The value passed to this parameter should exactly match the value bound to the above JNDI name.
+This section will not explain how to run liquibase from the command line, just how to add the required parameter. The parameter is ```document.storage.directory``` and should be added to the liquibase command in the form ```-Ddocument.storage.directory=/example/path```. The value passed to this parameter should exactly match the value bound to the system property above.
```

Comment:
@seanf What do you think of supporting the new property (so that we have a nice set of zanata-namespaced system properties), and internally adding some code to make sure that `document.storage.directory` gets populated with the value of the new property. Not sure if that's possible considering the order of execution of it all.

Alternatively, and since this will only be a problem when updating Zanata from a certain version X to this one, we could make it a documentation entry that states that when doing this massive upgrade, you might need to start the system with both properties.

Your thoughts?